dcos/subcommand.py

```diff
@@ -686,6 +653,69 @@ def package_json(self):
             return util.load_json(package_json_file)
 
 
+class Subproc():
+    """Represents a wrapper for subprocess
+    """
+
+    def __init__(self):
+        self._env = os.environ.copy()
+        lib_path = 'LD_LIBRARY_PATH'
+        lib_path_value = self._env.get(lib_path + '_ORIG')
+        # remove pyinstaller overriding `LD_LIBRARY_PATH`
+        # remvoe with stable release of fix:
+        # https://github.com/pyinstaller/pyinstaller/pull/2148
+        if lib_path_value is not None:
+            self._env[lib_path] = lib_path_value
+        elif self._env.get(lib_path):
+            del self._env[lib_path]
+
+        # Setuptools overrides path to executable from virtualenv,
+        # modify this so we can specify a different path
```

Comment:
Did we end up filing an issue against `setuptools` or asking them about the problem?